lib/inspec/secrets/yaml.rb

```diff
@@ -10,7 +10,8 @@ class YAML < Inspec.secrets(1)
 
     def self.resolve(target)
       unless target.is_a?(String) && File.file?(target) && ['.yml', '.yaml'].include?(File.extname(target).downcase)
-        return nil
+        puts "Warning file: #{target} was ignored, please use to .yml or .yaml"
```

Comment:
We cannot exit here. This code will also be used if another secrets backend is used.